### DISCUSSION TITLE
External IP connection attempts

### DISCUSSION THREAD

**Discussion by ll3N1GmAll** [Security-Onion-Solutions/securityonion#12394] [General]
We are seeing external IP contact attempts from launcher.exe to, at least, the following IP addresses:
208.111.186.0
208.111.186.128
35.196.217.93
34.149.84.181
23.218.218.71
2600:1408:c400:24::17da:d81b
2600:1408:c400:24::17da:d832
23.218.218.70
23.47.204.45
23.204.152.18
23.204.152.20
23.204.152.38
23.204.152.43
23.204.152.45
23.204.152.5
23.215.0.12
23.215.0.13
23.215.0.15
23.215.0.4
23.215.0.5
23.215.0.9
23.218.218.17
23.218.218.24
23.218.218.71
23.218.218.76
23.221.227.25
23.221.227.32
23.221.227.33
23.221.227.38
23.221.227.52
23.221.227.9
23.223.17.164
23.223.17.168
23.47.204.45
23.47.204.46
23.47.204.50
23.47.204.53
23.47.204.54
23.47.204.65
23.47.204.72
23.47.204.74
23.47.204.79
23.47.204.81
23.49.5.196
23.49.5.214
23.53.122.134
23.53.122.137
2600:1404:ec00:45::1724:d9af
2600:1404:ec00:45::1724:d9b0
2600:1407:b800::6872:4f91
2600:1407:b800::6872:4fba
2600:1408:c400:2e::17de:410
2600:1408:c400:2e::17de:41b
2600:1408:ec00:17::17d7:86
2600:1408:ec00:17::17d7:8f
2600:1408:ec00:23::1735:2349
2600:1408:ec00:23::1735:2350
2600:141b:f000:14::172e:9c86
2600:141b:f000:14::172e:9c9e
2600:141b:f000:14::172e:9ca9
2600:141b:f000:14::172e:9cb0
34.149.84.181
35.196.217.93
88.221.134.16
88.221.134.9
88.221.135.208
88.221.135.210
88.221.135.219
88.221.135.89
92.123.140.122
92.123.142.232

What are these attempts for? Is there any data from our network being sent anywhere? Thank you for your assistance.

**Comment by InfosecGoon** [comment]
I spot-checked a few of these against ARIN and they all appear to be Akamai IP space. Is launcher trying to update? Are you seeing any DNS queries or other metadata?
